src/driver/sqlite/SqliteConnectionOptions.ts

```diff
@@ -20,4 +20,8 @@ export interface SqliteConnectionOptions extends BaseConnectionOptions {
      */
     readonly key?: string;
 
-}
\ No newline at end of file
+    /**
+     * Enable ON CONFLICT clauses. (SQLITE >= v3.24.0)
+     */
+    readonly conflictSupport?: boolean;
```

Comment:
Think we don't need this flag. If users don't specify `this.expressionMap.onIgnore` or `this.expressionMap.onConflict` we don't break anything, if they specify, well they know what they are using

Reply:
My rationale behind the flag was maintaining the current behavior and avoid breaking changes. But I do agree that users should know what they are using.

I'll remove the flag.

Reply:
@pleerock Force-pushed the changes without the flag.